nunit.tests.vb/My Project/AssemblyInfo.vb

```diff
@@ -12,7 +12,7 @@ Imports System.Runtime.InteropServices
 <Assembly: AssemblyDescription("")> 
 <Assembly: AssemblyCompany("")> 
 <Assembly: AssemblyProduct("nunit.tests.vb")> 
-<Assembly: AssemblyCopyright("Copyright © 2017")> 
+<Assembly: AssemblyCopyright("Copyright © 2019")> 
```

Comment:
I'd be inclined to remove the AssemblyInfo for a test project template.

Reply:
If we used the new lightweight format, I agree, but with the old format - and if people want to have the "old" way, this is where they add that stuff.  If we change, we should have some way of telling them how they can add it anywhere, since this is a change in convention.

Reply:
To me this doesn't have anything to do with old vs new csproj format. Either way, I don't want this stuff specified (and always out of date) for test projects at all, whether in the .csproj or in an AssemblyInfo.cs file.

I think the project properties will put it back if users want it for the test project for some reason.

If we do include the file, can we leave it empty?

Reply:
I have removed it.